lib/routes/charge/items/item_page.dart

```diff
@@ -93,30 +119,25 @@ class CreateItemPageState extends State<CreateItemPage> {
                           child: TextFormField(
                               keyboardType: TextInputType.numberWithOptions(
                                   decimal: true),
-                              inputFormatters: _isFiat
+                              inputFormatters: _selectedCurrency.chargeSuffix ==
+                                      "BTC"
```

Comment:
Can we add the regex as property on currency? Call it "inputFormatter"?

Reply:
[Done](https://github.com/breez/breezmobile/pull/296/commits/de801c8a9c28fc66abb0abcce56d96aca4d6349f)